Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -573,6 +578,103 @@ namespace AzToolsFramework
+                        Path fromPath = item->GetFullPath();
+                        PathView filename = fromPath.Filename();
+                        Path toPath(folderPath);
+                        toPath /= filename;
+                        AssetChangeReportRequest request(
+                            AZ::OSString(fromPath.c_str()), AZ::OSString(toPath.c_str()), AssetChangeReportRequest::ChangeType::CheckMove);
+                        AssetChangeReportResponse response;
+
+                        if (SendRequest(request, response))
+                        {
+                            AZStd::string message;
+                            for (int i = 0; i < response.m_lines.size(); ++i)
+                            {
+                                message += response.m_lines[i] + "\n";
+                            }
+
+                            if (message.size())
+                            {
+                                QMessageBox msgBox(this);
+                                msgBox.setWindowTitle("Pre-Move Report");
+                                msgBox.setIcon(QMessageBox::Warning);
+                                msgBox.setText("The asset you are moving may be referenced in other assets.");
+                                msgBox.setInformativeText("More information can be found by pressing \"Show Details...\".");
+                                QAbstractButton* moveButton = (QAbstractButton*)msgBox.addButton("Move", QMessageBox::YesRole);
+                                msgBox.setStandardButtons(QMessageBox::Cancel);
+                                msgBox.setDefaultButton(QMessageBox::Yes);
+                                msgBox.setDetailedText(message.c_str());
+                                msgBox.exec();
+
+                                if (msgBox.clickedButton() == moveButton)
+                                {
+                                    AssetChangeReportRequest request2(
+                                        AZ::OSString(fromPath.c_str()),
+                                        AZ::OSString(toPath.c_str()),
+                                        AssetChangeReportRequest::ChangeType::Move);
+                                    AssetChangeReportResponse response2;
+                                    if (SendRequest(request2, response2))
+                                    {
+                                        AZStd::string message2;
+                                        for (int i = 0; i < response2.m_lines.size(); ++i)
+                                        {
+                                            message2 += response2.m_lines[i] + "\n";
+                                        }
+
+                                        if (message2.size())
+                                        {
+                                            QMessageBox msgBox2(this);
+                                            msgBox2.setWindowTitle("Move Report");
```

Comment:
Similar to above. If this would make sense to most users then disregard, but otherwise could it be more user-friendly?

Reply:
Changed to "After Move Asset Information".